app/views/welcome/approvals.html.erb

```diff
@@ -1,34 +1,46 @@
-<h1>Unapproved Users</h1>
+<div> 
+  <h1 class="mb-3 text-4xl text-gray-700">Unapproved Users</h1>
 
-<table class="table-auto">
+  <table class="w-full">
     <thead>
-        <tr>
-            <th>ID</th>
-            <th>Email</th>
-            <th>Balance</th>
-            <th>Confirmed?</th>
-            <th>Approved?</th>
-            <th>Actions?</th>
-        </tr>
-    </thead>    
+      <tr class="h-10 mb-5 text-sm font-semi-bold uppercase text-gray-600 bg-gray-300 text-center">
+        <th class="px-3">ID</th>
+        <th>Email</th>
+        <th>First Name</th>
+        <th>Last Name</th>
+        <th>Balance</th>
+        <th>Confirm Status</th>
+        <th>Approve</th>
+      </tr>
+    </thead>
 
     <tbody>
-        <% @unapproved_users.each do |user| %>
-            <tr>
-                <td><%= user.id %></td>
-                <td><%= user.email %></td>
-                <td><%= user.balance %></td>
-                <td><%= user.confirmed? %></td>
-                <td><%= user.approved %></td>
-                <td>
-                    <%=
-                        link_to 'Approve', 
-                        approve_admin_user_path(user, :user => {:approved => true}),
-                        :method => :put,
-                        data: { confirm: "Approve this user?" }
-                    %>
-                </td>
-            </tr>
-        <% end %>
+      <% @unapproved_users.each do |user| %>
+        <tr class="text-center h-16 <%= @toggle == "odd" ? "" : "bg-gray-200" %> ">
+          <% @toggle = (@toggle == "odd" ? "even" : "odd") %>
+          <td><%= user.id %></td>
+          <td><%= user.email %></td>
+          <td><%= user.first_name %></td>
+          <td><%= user.last_name %></td>
+          <%# <td><%= user.balance %> <%# </ td> %>
+          <td> $1000000 </td>
+          <td> 
+            <div class="w-full h-full flex justify-center items-center">
+              <div class="text-white rounded-3xl h-full w-20
+                <%= user.confirmed? ? "bg-green-600" : "bg-red-600" %> ">
+                <% if user.confirmed? %>
+                  <i class="fas fa-check"></i>
+                <% else %>
+                  <i class="fas fa-times"></i> 
+                <% end %>
+              </div> 
+            </div>
+          </td>
+          <td>
+            <%= link_to raw('<i class="far fa-check-circle text-blue-900 text-2xl"></i>'), approve_admin_user_path(user, :user => {:approved => true}), :method => :put, data: { confirm: "Approve this user?" } %>
```

Comment:
I used to do this, but I found a cleaner approach to putting HTML inside link_to!
https://stackoverflow.com/questions/11263968/how-to-pass-html-for-a-link-to-block/11264169